src/components/ServicesPage/Prices/index.tsx

```diff
@@ -0,0 +1,151 @@
+"use client";
+import React from "react";
+import SessionPriceCard from "@/components/ServicesPage/components/SessionPriceCard";
+import PhotographySection from "@/components/ServicesPage/components/PhotographySection";
+import { useState } from "react";
+
+const sections = [
+  {
+    title: "Portrait Photography",
+    description:
+      "Our portrait photography service is all about showcasing your unique personality. Whether you need a professional headshot, a family portrait, or a personal photoshoot, we create images that reflect your true self. We work closely with you to bring out your best angles and expressions, ensuring every portrait tells your story.",
+    linkText: "View Projects",
+    linkUrl: "/projects",
+    imageUrl: "/kytsya.jpg",
+  },
+];
+const sections2 = [
+  {
+    title: "Portrait Photography",
+    description:
+      "Our portrait photography service is all about showcasing your unique personality. Whether you need a professional headshot, a family portrait, or a personal photoshoot, we create images that reflect your true self. We work closely with you to bring out your best angles and expressions, ensuring every portrait tells your story.",
+    linkText: "View Projects",
+    linkUrl: "/projects",
+    imageUrl: "/kytsya.jpg",
+  },
+];
+const Prices: React.FC = () => {
+  const [currentSection, setCurrentSection] = useState(0);
+
+  const handlePrev = () => {
+    setCurrentSection((prev) => (prev > 0 ? prev - 1 : sections.length - 1));
+  };
+
+  const handleNext = () => {
+    setCurrentSection((prev) => (prev < sections.length - 1 ? prev + 1 : 0));
+  };
+
+  return (
+    <>
+      <div className="container mx-auto p-4">
+        <PhotographySection
+          title={sections[currentSection].title}
+          description={sections[currentSection].description}
+          linkText={sections[currentSection].linkText}
+          linkUrl={sections[currentSection].linkUrl}
+          imageUrl={sections[currentSection].imageUrl}
+          onPrev={handlePrev}
+          onNext={handleNext}
+        />
+        <div>
+          <SessionPriceCard
+            title="Individual Session"
+            price={250}
+            duration="2-hour"
```

Comment:
I think duration should be integer
So instead you would pass duration={2} and inside the component add {duration}-hour